### PR TITLE
257 rtl fixes

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

File: frontend/src/components/Appbar.tsx [hunk 1]
```diff
@@ -100,7 +104,7 @@ const Appbar: FC<AppbarProps> = (props) => {
                 sx={{ padding: '0 1rem' }}
               >
                 <StepButton sx={{ padding: '0', margin: '0' }} disableRipple>
-                  <StepLabel>{step}</StepLabel>
+                  <StepLabel StepIconProps={{ sx: { paddingX: '1rem' } }}>{step}</StepLabel>
```
Comment (avirankatz) [inline frontend/src/components/Appbar.tsx:107]:
```suggestion
                  <StepLabel StepIconProps={{ sx: { paddingX: 1 } }}>{step}</StepLabel>
```

**Review by avirankatz — CHANGES_REQUESTED** [review]

File: frontend/src/components/Appbar.tsx (hunk 1 above)
Comment (yishain11) [reply frontend/src/components/Appbar.tsx:107]: done

File: frontend/src/pages/Agreement.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ const Agreement: FC = () => {
       <Appbar breadcrumbs={breadcrumsProps} />
       <Stack spacing={4} paddingX={2} paddingY={3}>
         <Stack direction="row" alignItems="center" justifyContent="space-between">
-          <Stack direction="row" alignItems="end">
+          <Stack direction="row" alignItems="end" gap={0.5}>
             <Typography variant="h1" marginRight={2}>
```
Comment (nadavmoreno) [inline frontend/src/pages/Agreement.tsx:71]: @yishain11 remove `marginRight`

File: frontend/src/pages/Agreement.tsx [hunk 3]
```diff
@@ -67,7 +67,7 @@ const Agreement: FC = () => {
       <Appbar breadcrumbs={breadcrumsProps} />
       <Stack spacing={4} paddingX={2} paddingY={3}>
         <Stack direction="row" alignItems="center" justifyContent="space-between">
-          <Stack direction="row" alignItems="end">
+          <Stack direction="row" alignItems="end" gap={0.5}>
```
Comment (nadavmoreno) [inline frontend/src/pages/Agreement.tsx:70]: @yishain11 change `gap` to `1.5`

**Pull request merged**